backend/src/dto/todoDto.ts

```diff
@@ -1,22 +1,23 @@
-import { IsString, IsIn, IsInt, IsUUID } from 'class-validator';
-const todoStatusValues = ['pending', 'completed'] as const;
-const todoStringKeys = ['title', 'date'] as const;
-const todoStatusKey = ['status'] as const;
-export type TodoStringKeyTypes = (typeof todoStringKeys)[number];
-export type statusValues = (typeof todoStatusValues)[number];
+import { IsString, IsIn, IsUUID } from 'class-validator';
+const statusValues = ['pending', 'completed'] as const;
+export type StatusValuesTypes = (typeof statusValues)[number];
```

Comment:
no need for the `Types` in the name, you are defining a type so of course it is a type
also let's rename both of them to:
```
const todosStatuses = ['pending', 'completed'] as const;
export type TodoStatus = (typeof todosStatuses)[number];
```